package.json

```diff
@@ -17,6 +17,8 @@
     "typescript": "^5.7.2"
   },
   "dependencies": {
-    "haveno-ts": "^0.0.30"
+    "decimal.js": "^10.4.3",
```

Comment:
Was hoping / expecting that these dependencies are bundled with haveno-ts, so dependent apps don't have to install multiple dependencies.  Possible?

Reply:
Yes. Possible. Will refactor.